package.json

```diff
@@ -37,7 +38,10 @@
     "@typescript-eslint/parser": "^5.30.5",
     "@vitejs/plugin-react": "^1.3.2",
     "@vitest/ui": "^0.18.0",
+    "aws-cdk-lib": "^2.145.0",
```

Comment:
It's recommended to configure deployment environment in the separate folder to avoid conflicts between project's and deployment's dependencies.